react/components/Tabs/index.test.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { toHaveClass, toHaveAttribute, toHaveTextContent } from '@testing-library/jest-dom/matchers'
+
+import Tabs from '.'
+import Tab from './Tab.js'
+
+describe('Tabs', () => {
+  expect.extend({ toHaveClass, toHaveAttribute, toHaveTextContent })
+  const tabsId = 'tabs'
+  const tab1Id = 'tab 1'
+
+  describe('CSS API', () => {
+
+    it('props: default', () => {
+      const { getByTestId } = render(
+        <Tabs testId={tabsId}>
+          <Tab testId={tab1Id} label="label 1" active>
+            Tab 1
+          </Tab>
+          <Tab label="label 2" />
+        </Tabs>
+      )
+      
+      const tabs = getByTestId(tabsId)
+      expect(tabs).not.toHaveClass('overflow-y-hidden')
+      
+      const tabsContent = tabs.querySelector('.overflow-y-auto')
+      expect(tabsContent).toBeNull()
+      expect(tabsContent).toBeDefined()
+
+      expect(getByTestId(tab1Id)).not.toHaveClass('w-100')
+    })
+    
+    it('prop: sticky', () => {
+      const { getByTestId } = render(
+        <Tabs testId={tabsId} sticky>
+          <Tab label="label 1" active>
+            Tab 1
+          </Tab>
+          <Tab label="label 2" />
+        </Tabs>
+      )
+
+      const tabs = getByTestId(tabsId)
+      expect(tabs).toHaveClass('overflow-y-hidden')
+      
+      const tabsContent = tabs.querySelector('.overflow-y-auto')
+      expect(tabsContent).not.toBeNull()
+      expect(tabsContent).toBeDefined()
+    })
+
+    it('prop: fullWidth', () => {
+      const { getByTestId } = render(
+        <Tabs testId={tabsId} fullWidth>
+          <Tab testId={tab1Id} label="label 1" active>
+            Tab 1
+          </Tab>
+          <Tab label="label 2">Tab 2</Tab>
+          <Tab label="label 3">Tab 3</Tab>
+        </Tabs>
+      )
+
+      expect(getByTestId(tabsId)).toHaveClass('w-100')
+      expect(getByTestId(tab1Id)).toHaveClass('w-100')
+    })
+  })
+
+  describe('Behavior', () => {
+    it('active tab content', () => {
+      const { getByTestId } = render(
+        <Tabs testId={tabsId}>
+          <Tab label="label 1" active>
+            Tab 1
+          </Tab>
+          <Tab label="label 2">Tab 2</Tab>
+        </Tabs>
+      )
+
+      expect(getByTestId(tabsId)).toHaveTextContent('Tab 1')
```

Comment:
[Avoid using getByTestId when possible](https://testing-library.com/docs/guide-which-query)

```suggestion
      expect(getByText('Tab 1')).toBeTruthy()
```

Reply:
Nice @klzns, I will rethink about the way I did my tests.

Reply:
but you don't need to remove the `testId` prop from your component, test only.

Reply:
I removed from tab because if I want to get it, I can use getByText that's suggested by `Testing Library`